docs/contributing/docs-contributions/how-to-write-a-plugin-readme.md

```diff
@@ -3,6 +3,8 @@ title: How to Write a Plugin README
 issue: https://github.com/gatsbyjs/gatsby/issues/21599
```

Comment:
Is this still needed?

Reply:
Honestly, I left it because I'm not sure what this frontmatter does, and was mainly focused on content this time through 😅

Reply:
Do you know what it's for? I don't mind deleting it!

Reply:
afaik it's for displaying a note about the stub issue. See at the bottom here: https://www.gatsbyjs.com/contributing/how-to-write-a-plugin-readme/

> See the issue relating to this stub on GitHub

So if we deleted all stubs then we can get rid of these `issue` in frontmatter

Reply:
Gotcha, thanks! I'll delete it ✂️

Reply:
(Hm that didn't work the way I thought it would. That's what I get for using the UI.)